src/aap_eda/api/views/activation.py

```diff
@@ -79,6 +83,14 @@ def create(self, request):
         )
         serializer.is_valid(raise_exception=True)
 
+        # If we're expected to run this activation we need redis
+        # to be available.
+        if serializer.validated_data.get(
+            "is_enabled",
+            models.activation.DEFAULT_ENABLED,
```

Comment:
is this line correct? Should it be `models.activation.is_enabled`?

Reply:
@Dostonbek1 This is accessing the activation.py package for the default value as there's no Activation instance at this point and the request may not actually contain is_enabled.
Only after calling serializer.create() would we be guaranteed of having is_enabled (on the created activation) but I wanted to avoid doing anything with the DB (even though it's wrapped in a transaction).

I considered this an area where I might get "just do it in the transaction" so I'm prepared to change if people want. :-)